README.md

```diff
@@ -1,10 +1,14 @@
 # DustExtinction.jl
 
 [![Build Status](https://img.shields.io/travis/JuliaAstro/DustExtinction.jl.svg?style=flat-square)](https://travis-ci.org/JuliaAstro/DustExtinction.jl)
+[![](https://img.shields.io/badge/docs-stable-blue.svg)](https://JuliaAstro.github.io/DustExtinction.jl/stable)
+[![](https://img.shields.io/badge/docs-dev-blue.svg)](https://JuliaAstro.github.io/DustExtinction.jl/dev)
```

Comment:
The link should be to https://JuliaAstro.github.io/DustExtinction.jl/latest, not dev

Reply:
I copy and pasted those directly from the Documenter documentation, https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#Hosting-Documentation-1

Reply:
Strange, I was looking to other docs build with Documenter in JuliaAstro:

* http://juliaastro.github.io/FITSIO.jl/latest/
* http://juliaastro.github.io/AstroLib.jl/latest/

Well, we can keep this and will see where the documentation goes

Reply:
Uhm, probably newer versions of Documenter changed the link to `dev`